src/lib/transports/splunkHEC.js

```diff
@@ -25,8 +38,13 @@ class SplunkHEC extends winston.Transport {
 			},
 			pool: httpsAgent,
 			timeout: 3000,
-			body: inspect(data)
-		}).catch(() => {});
+			body: stringify(data)
+		})
+			.then(throwIfNotOk)
+			.catch((error) => {
+				/* eslint no-console: 0 */
+				console.log('Error logging to splunk', error, data);
```

Comment:
This feels like it would easily get lost... is it worth maybe checking for the existence of `process.env.RAVEN_URL` and logging to sentry or `process.env.FT_GRAPHITE_APP_UUID` for Grafana which could have a healthcheck associated with it?

Reply:
After some discussion with @remybach, this seems like a bigger issue that needs further investigation. I've added an issue to look into it here: https://github.com/Financial-Times/n-logger/issues/83